examples/modeling_features/021-imported-solid-model.py

```diff
@@ -0,0 +1,275 @@
+In contrast to the :class:`.SolidModel`, the raw solid mesh of
+:class:`.ImportedSolidModel` is loaded from an external source, such as a CDB file.
+In this example, the layup is applied onto a t-joint which consists of different
+parts such as shell, stringer, and bonding skins.
+The example only shows the PyACP part of the setup. For a complete composite analysis,
+see :ref:`pymapdl_workflow_example`.
+
+This example starts from an ACP model with layup. It shows how to:
+
+- Create an :class:`.ImportedSolidModel` from an external mesh.
+- Define the :class:`.LayupMappingObject` to apply the layup onto the solid mesh.
+- Scope plies to specific parts of the solid mesh.
+- Visualize the mapped layup.
+
+It is recommended to look at the Ansys help for all the details. This example shows the
+basic setup only.
+"""
+
+# %%
+# Import the standard library and third-party dependencies.
+import pathlib
+import tempfile
+
+import pyvista
+
+# %%
+# Import the PyACP dependencies.
+from ansys.acp.core import (
+    ACPWorkflow,
+    ElementTechnology,
+    LayupMappingRosetteSelectionMethod,
+    launch_acp,
+)
+from ansys.acp.core.extras import ExampleKeys, get_example_file
+
+# sphinx_gallery_thumbnail_number = 3
+
+
+# %%
+# Start ACP and load the model
+# ----------------------------
+# %%
+# Get the example file from the server.
+tempdir = tempfile.TemporaryDirectory()
+WORKING_DIR = pathlib.Path(tempdir.name)
+input_file = get_example_file(ExampleKeys.IMPORTED_SOLID_MODEL_ACPH5, WORKING_DIR)
+
+# %%
+# Launch the PyACP server and connect to it.
+acp = launch_acp()
+
+# %%
+# Define the input file and instantiate an ``ACPWorkflow`` instance.
+workflow = ACPWorkflow.from_acph5_file(
+    acp=acp,
+    acph5_file_path=input_file,
+    local_working_directory=WORKING_DIR,
+)
+
+model = workflow.model
+
+# %%
+# Import external solid model
+# ---------------------------
+#
+# Get the solid mesh file and create an ImportedSolidModel and
+# load the initial mesh.
+
+local_solid_mesh_file = get_example_file(ExampleKeys.IMPORTED_SOLID_MODEL_SOLID_MESH, WORKING_DIR)
+remote_solid_mesh_file = acp.upload_file(local_solid_mesh_file)
+imported_solid_model = model.create_imported_solid_model(
+    name="Imported Solid Model",
+    external_path=remote_solid_mesh_file,
+    format="ansys:h5",
+)
+
+# Load the initial mesh and show the raw mesh without any mapping.
+imported_solid_model.refresh()
+imported_solid_model.import_initial_mesh()
+model.solid_mesh.to_pyvista().plot(show_edges=True)
+
+# %%
+# The solid element sets can be used as target for the mapping.
+print(imported_solid_model.solid_element_sets)
+
+# %%
+# Add mapping objects
+# -------------------
+#
+# Link the layup (plies) of the top skin of the sandwich
+# with the corresponding named selections of the solid mesh
+# and show the updated solid model.
+solid_esets = imported_solid_model.solid_element_sets
+
+_ = imported_solid_model.create_layup_mapping_object(
```

Comment:
So that the return value is not printed when running the example.

Reply:
This only happens for the _last_ statement in the "cell" (i.e., directly before another `# %%`), right?

Reply:
I tried to figure out if this can be suppressed on a per-cell basis using sphinx-gallery inline options. Didn't find an option, though. Here's the relevant docs: https://sphinx-gallery.github.io/stable/configuration.html#controlling-what-output-is-captured

Reply:
Yes, this happens only for the last statement. It's not critical though and so we can remove discarding the return value if you think that is better.

Reply:
I don't have a _very_ strong opinion on it, but think the `_ =` might be slightly confusing for the reader. So I'd tend towards removing it if the generated output is acceptable.